pom.xml

```diff
@@ -62,7 +62,7 @@
         <componentGroupName>WKND Site</componentGroupName>
         <core.wcm.components.version>2.16.4</core.wcm.components.version>
         <uber.jar.version>6.4.4</uber.jar.version>
-        <aemanalyser.version>0.9.2</aemanalyser.version>
+        <aemanalyser.version>1.1.4</aemanalyser.version>
```

Comment:
The latest version is 1.1.8, so it would be best to use that instead: https://repo1.maven.org/maven2/com/adobe/aem/aemanalyser-maven-plugin/1.1.8/